### PR TITLE
Add instructions for the Windows SubSystem Linux

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: README_Windows.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+       1. Install Python with the following: 
+
+         `apt -y install python`
+
+       1. Install Ruby with the following:
+
+          ```zsh
+          cd
+          git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+          echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.zshrc
```
Comment (sandbergja) [inline README_Windows.md:27]: BASH, rather than ZSH was installed in WSL for me.

File: README_Windows.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          echo 'eval "$(rbenv init -)"' >> ~/.zshrc
+          exec $SHELL
+
+          git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+          echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.zshrc
+          exec $SHELL
+
+          rbenv install 3.1.3
```
Comment (sandbergja) [inline README_Windows.md:35]: I needed to `sudo apt-get install -y gcc make zlib1g-dev` first to make this step succeed.

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks @kayiwa !  I added a few notes based on my experience in WSL on Windows 11.

File: README_Windows.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          rbenv global 3.1.3
+
+          gem install bundler
+
+          rbenv rehash
+          ```
+       1. `pip install --upgrade pipenv`
```
Comment (sandbergja) [inline README_Windows.md:42]: I needed to `sudo apt -y install python3-pip` for this step to work.

**Comment by kayiwa** [comment]
thanks so much for testing @sandbergja incorporated your changes.

**Review by sandbergja — APPROVED** [review]
Thanks, @kayiwa !

**Pull request merged**